packages/actionmenu/package.json

```diff
@@ -21,10 +21,13 @@
     "test:watch": "yarn test --watchAll"
   },
   "exports": {
-    "import": "./dist/esm/index.mjs",
-    "require": "./dist/cjs/index.js",
-    "node": "./dist/cjs/index.js",
-    "default": "./dist/cjs/index.js"
+    ".": {
```

Comment:
what is this witchcraft

Reply:
It is a path. It means, when importing, "the package name, with no paths beyond it". So, this case:

`import Button from '@pluralsight/ps-design-system-button'

https://nodejs.org/api/packages.html#packages_subpath_exports

If you don't have multiple paths, you can assume the default "no path" path, which is what we had. 

In other words, yes, witchcraft.